opentracing-spring-cloud/src/main/resources/META-INF/spring.factories

```diff
@@ -1,10 +1,12 @@
 org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
 io.opentracing.contrib.spring.cloud.feign.FeignTracingAutoConfiguration,\
+io.opentracing.contrib.spring.cloud.feign.FeignContextBeanPostProcessor,\
```

Comment:
Do `FeignContextBeanPostProcessor` and `ExecutorBeanPostProcessor` have to be defined in separate conf files?

Reply:
Yes it appears that need to be defined separately.

Reply:
Correction - might be possible - will commit some changes soon.